server/server.go

```diff
@@ -135,7 +139,8 @@ func (cs *cribbageServer) ginPostCreatePlayer(c *gin.Context) {
 		}
 		return
 	}
-	c.JSON(http.StatusOK, player)
+	// TODO do we really need to return the player?
```

Comment:
Yes, it's helpful for the frontend, and eventually when we have real auth there's probably a token or something we'll want to return here too

Reply:
nice

Reply:
Actually, we could probably pivot to returning a "create player response"